spec/features/dance_comments/3_user_creates_spec.rb

```diff
@@ -41,9 +41,8 @@
     click_link "Create Comment"
     click_button "Create Dance comment"
 
-    expect(page).to have_content(
-      "Comment can't be blank"
-    )
+    expect(page).to have_content("Comment can't be blank")
+
     # CANNOT SEEM TO TEST FOR TYPE CAN'T BE BLANK.
```

Comment:
Hmm, your associated code looks good. Perhaps the type error is being produced but is not being matched by the `expect` you had written? I would def reach out to chat support to troubleshoot this together.